rustls/src/manual/features.rs

```diff
@@ -59,4 +59,39 @@ and will not support:
   path building. While dangerous, all authentication can be turned off if required --
   see the [example code](https://github.com/rustls/rustls/blob/992e2364a006b2e84a8cf6a7c3eaf0bdb773c9de/examples/src/bin/tlsclient-mio.rs#L318)_ `*`
 
+### About "custom extensions"
+
+OpenSSL allows an application to add arbitrary TLS extensions (via
+the `SSL_CTX_add_custom_ext` and associated APIs).  We don't support this,
```

Comment:
editorial nit: "via the `SSL_CTX_add_custom_ext` and associated APIs" -> "via the `SSL_CTX_add_custom_ext` function and associated APIs"